src/main/java/com/softserve/actent/controller/MessageController.java

```diff
@@ -32,46 +29,55 @@
 
     private final MessageService messageService;
 
+    private final UserService userService;
+
+    private final ChatService chatService;
+
     private final ViewMessageConverter viewMessageConverter;
 
-    private final TextMessageConvert textMessageConvert;
 
     private final ModelMapper modelMapper;
 
-    private final ImageMessageConverter imageMessageConverter;
-
     @Autowired
     public MessageController(MessageService messageService,
                              ViewMessageConverter viewMessageConverter,
-                             TextMessageConvert textMessageConvert, ModelMapper modelMapper, ImageMessageConverter imageMessageConverter) {
+                             ModelMapper modelMapper,
+                             UserService userService,
+                             ChatService chatService) {
+
         this.messageService = messageService;
         this.viewMessageConverter = viewMessageConverter;
-        this.textMessageConvert = textMessageConvert;
         this.modelMapper = modelMapper;
-        this.imageMessageConverter = imageMessageConverter;
+        this.userService = userService;
+        this.chatService = chatService;
     }
 
 
     @PostMapping(value = "/textMessages")
     public ResponseEntity<ViewMessageDto> addMessage(@RequestBody CreateTextMessageDto createMessageDto) {
 
-        Message message = messageService.add(textMessageConvert.convertToEntity(createMessageDto));
+        Message message = modelMapper.map(createMessageDto, Message.class);
+        message.setSender(userService.get(createMessageDto.getSenderId()));
+        message.setChat(chatService.getChatById(createMessageDto.getChatId()));
 
-        return new ResponseEntity<>(viewMessageConverter.convertToDto(message), HttpStatus.CREATED);
+        return new ResponseEntity<>(viewMessageConverter.convertToDto(messageService.add(message)), HttpStatus.CREATED);
```

Comment:
the same as in Ira's merge. 
@RestController and return new ResponseEntity read about that. (+ the difference between @RestController  and @Controller)

and, please, review and communicate with each other to not to repeat the same mistakes